openziti-init-entrypoint.sh

```diff
@@ -0,0 +1,80 @@
+  ziti edge create edge-router-policy "${svc}.erp" --identity-roles "#${svc}" --edge-router-roles '#all'
+  ziti edge create service-policy "${svc}.bind" Bind --service-roles "#${svc}" --identity-roles "#${svc}-servers"
+  ziti edge create service-policy "${svc}.dial" Dial --service-roles "#${svc}" --identity-roles "#${svc}-clients"
+}
+
+sleep 1
+makeEdgexService 'core-command'
+makeEdgexService 'core-data'
+makeEdgexService 'core-metadata'
+makeEdgexService 'device-virtual'
+makeEdgexService 'device-rest'
+makeEdgexService 'rules-engine'
+makeEdgexService 'support-notifications'
+makeEdgexService 'support-scheduler'
+makeEdgexService 'sys-mgmt-agent'
+makeEdgexService 'ui'
+makeEdgexService 'app-rules-engine'
+
+
+ziti edge create identity edgexuser -P "${edgex_auth_policy}" --external-id edgexuser -a edgex.device-virtual-servers,edgex.core-command-clients,edgex.core-data-clients,edgex.core-metadata-clients,edgex.device-virtual-clients,edgex.rules-engine-clients,edgex.support-notifications-clients,edgex.support-scheduler-clients,edgex.sys-mgmt-agent-clients
+ziti edge update identity edgex.rules-engine -a edgex.rules-engine-servers,edgex.core-command-clients
+ziti edge update identity edgex.device-virtual -a edgex.device-virtual-servers,edgex.core-command-clients,edgex.core-data-clients,edgex.core-metadata-clients,edgex.device-virtual-clients,edgex.rules-engine-clients,edgex.support-notifications-clients,edgex.support-scheduler-clients,edgex.sys-mgmt-agent-clients
+ziti edge update identity edgex.ui -a edgex.device-virtual-servers,edgex.core-command-clients,edgex.core-data-clients,edgex.core-metadata-clients,edgex.device-virtual-clients,edgex.rules-engine-clients,edgex.support-notifications-clients,edgex.support-scheduler-clients,edgex.sys-mgmt-agent-clients
+ziti edge update identity edgex.device-rest -a edgex.device-rest-servers,edgex.core-command-clients,edgex.core-data-clients,edgex.core-metadata-clients,edgex.device-virtual-clients,edgex.rules-engine-clients,edgex.support-notifications-clients,edgex.support-scheduler-clients,edgex.sys-mgmt-agent-clients
+ziti edge update identity edgex.core-command -a edgex,edgex.core-command-servers,edgex.core-metadata-clients,edgex.device-virtual-clients
+
+echo "granting additional dial capabilities to app-rules-engine"
+ziti edge update identity edgex.app-rules-engine -a edgex,edgex.app-rules-engine-servers,edgex.core-metadata-clients
+
+echo " "
+echo "ext-jwt-id     : ${ext_jwt_id}"
+echo "auth policy id : ${edgex_auth_policy}"
+echo " "
+
+echo "creating the healthcheck proxy identity"
+ziti edge create identity \
+  healthcheck -o ${OPENZITI_PERSISTENCE_PATH}/healthcheck.jwt \
+  -a 'edgex.app-rules-engine-clients,edgex.device-rest-clients,edgex.core-command-clients,'\
+'edgex.core-data-clients,edgex.core-metadata-clients,edgex.device-virtual-clients,'\
+'edgex.rules-engine-clients,edgex.support-notifications-clients,edgex.support-scheduler-clients,'\
+'edgex.sys-mgmt-agent-clients'
```

Comment:
Again we need to move this into `makeEdgexService` so we don't have to list out every service again.